### PR TITLE
Create home page

### PR DESCRIPTION
**Pull request by MenendezON**
In this pull request, we have 3 parts :
- The first part talks about who I'm
- The second part shows up some of my projects done
- The third part talks about my skills and my social accounts network

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,212 @@
...
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/MenendezON">
+                            <img src="images/github.png" alt="" srcset="">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </article>
```
Comment (AbayomiOlaoye) [inline index.html]: - [ ] Great job with this amazing improvisation, it looks really nice :clap: However, this project was intended to put certain skills and knowledge to use, which could help strengthen your design skills with perfectly transforming a Figma wireframe into nicely designed web app/page.
This is what it looks like when compared now:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|
- [x] Kindly keep strictly to the design in Figma (your logo can be retained, it's really beautiful)
- [x] The `colors` of all your texts - `p`, `h1` are not same with Figma.
- [x] Also, please add the social media icons, so you could be easily contacted if your users intend to. This also makes it match the design :wink:

File: index.html [hunk 2]
```diff
@@ -1,12 +1,212 @@
...
+                        <ul>
+                            <li>Project management</li>
+                            <li>IT Network</li>
+                            <li>Problem-solving</li>
+                        </ul>
+                    </div>
+                </aside>
+            </div>
+        </section>
+    </section>
```
Comment (AbayomiOlaoye) [inline index.html]: - [x] Good job with the additional requirements but really, this is for future milestones :+1:

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @MenendezON :wave:, 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MenendezON) [reply index.html]: Thank you Sir

File: index.html [hunk 3]
```diff
@@ -1,12 +1,212 @@
...
+                    </li>
+                    <li>
+                        <a href="https://github.com/MenendezON">
+                            <img src="images/github.png" alt="" srcset="">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </article>
+    </section>
```
Comment (Qoosim) [inline index.html]: - [ ] Great job implementing requested changes by the previous reviewer. However, there are few things you need to do before your project could be approved. 
- 1  There's background image on the figma design which is absent on your design. Kindly add that background image. Also remember that it should be added through css `background-image` property.
- 2  You must also pay attention to how each element in this section is design. This includes font-style, font-size, font-weight, font-type, color, padding, margin among others. Kindly look into this and fix.

### Your Design
your-headline

### Figma Design
temp-headline

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @MenendezON, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -52,237 +23,90 @@ After you're finished please remove all the comments and instructions!
...
-
-## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
```
Comment (Lookingfor00) [inline README.md]: - [x] **[OPTIONAL]** Please consider adding back **Contribution** and **Show your support** sections. Document your readme file will make your project look much more professional 😊

File: style.css [hunk 5]
```diff
@@ -1,4 +1,323 @@
...
+
+.intro {
+  height: 85%;
+  padding: 25px;
+  background-color: white;
+  border-bottom-left-radius: 75px;
+}
+
+.bgImage {
+  background-image: url(/images/imgbg-mobile.jpeg);
```
Comment (Lookingfor00) [inline style.css:181]: - [x] Please consider fixing the background image of your headline section as mentioned by the previous reviewer. You can do that by fixing the image URL path. I would suggest: `background-image: URL (./images/imaging-mobile.jpeg);`. Doing that will make your project look way prettier which will improve the user experience 😄 
**Your project headline still have no background**
image

**Review by Trast00 — CHANGES_REQUESTED** [review]
# Require Change ♻️ 
Hi @MenendezON , 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there-reach

## Highlights
- Excellent work for going beyond the current requirement and implementing the mobile interaction 😮 👏 👏 🥇  
- Great job implementing all changes requested by the previous reviewers
- Professional documented project 📖
- Great job adding your own logo 👽 

## Required Changes ♻️
_Check the comments under the review._

- Please consider implementing all the change I left below (inline comment)
- [ ] **[OPTIONAL]** Please consider updating your README.md file so meet some requirement missed by the previous reviewer (More information in inline comment). Note that missing these requirement may cause INVALIDE PR. Document your readme file will make your project look much more professional 😊 and help you show a great documentation skill 😄

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@Trast00) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (MenendezON) [reply style.css:181]: Done @Trast00

File: README.md (hunk 4 above)
Comment (MenendezON) [reply README.md]: Done! @Trast00

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**